### DISCUSSION TITLE
Method Spatie\Analytics\Analytics::performQuery does not exist.

### DISCUSSION THREAD

**Discussion by chiragwebiots** [freekmurze/laravel-analytics#479] [Q&A] [answered]

**Comment by luisscruza** [comment]
Same problem.

**Comment by tominal** [comment]
Hey @chiragwebiots, did you find a resolution to this?

**Comment by jeanquark** [comment, marked as answer]
Try using get function (see README for details):
`public function get(Period $period, array $metrics, array $dimensions = [], int $limit = 10, array $orderBy = []): Collection`
